TravelPoints-BackEnd/src/main/java/com/example/travelpointsbackend/model/auth/AuthenticationResponse.java

```diff
@@ -0,0 +1,15 @@
+package com.example.travelpointsbackend.model.auth;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class AuthenticationResponse {
+    private String token;
+}
```

Comment:
S: you should add the user id (from the db) here because you are going to have requests that will need it

Reply:
The ID is encoded in the token.

Reply:
you will have to synchronize with @ancachetan and @dragusandreea so that they know how to decode it on UI side. Instead of doing more front end processing I always prefer to send a JSON that looks like this: `{id: userId,  role: userRole, token: userToken}` .